src/command.js

```diff
@@ -27,11 +29,29 @@ const Command = class {
 
   static create (rawInputs) {
     const command = doNotAllowMissingProperties(new this(rawInputs))
+    this._sanitizeCreation(command)
     this._validateCreation(command)
+
     return command
   }
 
+  // If there are extra validations, put them here.
   static _validateCreation (newlyCreatedCommandInstance) {
+    this.validateThatSchemaMustBeStatic(newlyCreatedCommandInstance)
+  }
+
+  // If there are extra sanitization processes, put them here.
+  static _sanitizeCreation (newlyCreatedCommandInstance) {
+    this._sanitizeEmptySchema(newlyCreatedCommandInstance)
+  }
+
+  static validateThatSchemaMustBeStatic (newlyCreatedCommandInstance) {
+    if (newlyCreatedCommandInstance._staticSchemaAttributeWasOverwritten) {
+      throw new CommandWithNonStaticSchemaError('Schema must be a static property -- not an instance property.')
+    }
+  }
+
+  static _sanitizeEmptySchema (newlyCreatedCommandInstance) {
     if (isNull(newlyCreatedCommandInstance.schema) || isUndefined(newlyCreatedCommandInstance.schema)) {
       newlyCreatedCommandInstance.constructor.schema = {}
```

Comment:
This can be read as well like:
`Command.schema = { }` -- Which is an assignment to the static variable.

Reply:
Mentioned in the other PR, but I think we shouldn't mutate the schema here in an instance method. That seems kind of surprising to me. We could play with this and see if we can just give a default overridable empty static schema